### PR TITLE
New sauce

### PR DESCRIPTION
**Pull request by vweevers**
*Do not merge yet.*

**Changes:**

- New credentials
- Use [Sauce Connect](https://docs.travis-ci.com/user/sauce-connect/) instead of localtunnel
- Use [JWT addon](https://docs.travis-ci.com/user/jwt/)
- Enable debug output for zuul (on this flaky platform any info is welcome)

**Pros:**

- Sauce Connect is *probably* more reliable
- With Sauce Connect + JWT, PRs can also use Sauce Labs 💯 

**Cons:**

- Travis starts Sauce Connect in each job (also the node jobs), adding ~20 seconds to each
- Sauce Connect needs [a workaround for Safari and Edge](https://support.saucelabs.com/hc/en-us/articles/115002212447-Unable-to-Reach-Application-on-localhost-for-Tests-Run-on-Safari-8-and-9-and-Edge). Addressed by c05bff0d52c87f780555827ff77e5573c2bb0e6a.

**Need to:**

- [x] rebase and squash
- [x] make a PR to zuul (or temporarily drop Safari and Edge)
- [x] I want to try running all browsers in one travis job, see if this does work with Sauce Connect

### CODE REVIEW

**Comment by vweevers** [comment]
See also the thread starting here: https://github.com/Level/memdown/pull/79#issuecomment-333301801

**Comment by vweevers** [comment]
Pros of a single job for browser tests (f7879d5):

- Travis doesn't start Sauce Connect for non-browser tests, in fact, it only starts SC once
- Reduces build time from 30 to 9 minutes 💯 

Cons:

- Travis log is hard to read (interspersed results of multiple browsers), but we can view individual browsers in the Sauce Labs dashboard.

**Comment by vweevers** [comment]
Restarting the build just to test: https://travis-ci.org/Level/memdown/builds/281860132

**Comment by vweevers** [comment]
@ralphtheninja it *looks like* the browser tests are actually stable now.

**Comment by vweevers** [comment]
Depends on defunctzombie/zuul#323.

**Comment by ralphtheninja** [comment]
@vweevers Fuckin' A! So we wait until `zuul` has merged your fix?

**Comment by vweevers** [comment]
Yea, but I'm not sure it's still actively maintained..

**Comment by ralphtheninja** [comment]
> Yea, but I'm not sure it's still actively maintained..

Worst case scenario, we can use your fork for now.

**Comment by ralphtheninja** [comment]
@vweevers 

```
$ npm owner ls zuul
defunctzombie <shtylman@gmail.com>
feross <feross@feross.org>
vvo <vincent.voyer@gmail.com>
```

Seems @feross can help us out if @defunctzombie doesn't have time

File: .zuul.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - name: ie
+    version: latest
+  - name: microsoftedge
+    version: latest
+  - name: iphone
+    version: latest
+  - name: ipad
+    version: latest
+  - name: android
+    version: latest
```
Comment (ralphtheninja) [inline .zuul.yml]: :+1:

**Comment by vweevers** [comment]
Awesomesauce 😁

**Comment by ralphtheninja** [comment]
@vweevers I vote for merging this now and worry about getting your fork into upstream later.

**Pull request merged**